packages/compass-schema/src/components/field/field.jsx

```diff
@@ -119,7 +120,7 @@ class Field extends Component {
    */
   getSemanticType(type) {
     // check if the type represents geo coordinates, if privacy settings allow
-    if (global.hadronApp.isFeatureEnabled('enableMaps')) {
+    if (this.props.enableMaps) {
```

Comment:
Similar question here – instead of adding another variable to the store, installing a listener etc., can we just rewrite this check here?

Reply:
Also updated this.